windows/test-app.js

```diff
@@ -149,53 +147,61 @@ function copyProjectTemplateAndReplace(
   );
 
   fs.mkdirSync(projectFilesDestPath, { recursive: true });
-  fs.mkdirSync(path.join(destPath, windowsDir), { recursive: true });
+  fs.mkdirSync(destPath, { recursive: true });
 
   const manifestFilePath = findClosestPathTo("app.json");
 
   //Read path to resources from manifest
   let bundleDirContent = "";
   let bundleFileContent = "";
-  const content = fs.readFileSync(manifestFilePath);
-  let resourcesPaths = {};
-  try {
-    resourcesPaths = JSON.parse(content.toString()).resources;
-  } catch (e) {
-    console.warn(`Couldn't parse app.json: \n${e.message}`);
-  }
-  resourcesPaths = (resourcesPaths && resourcesPaths.windows) || resourcesPaths;
-  if (Array.isArray(resourcesPaths)) {
-    for (const resource of resourcesPaths) {
-      const resourceSrcPath = path.relative(
-        path.dirname(manifestFilePath),
-        resource
-      );
-      if (fs.existsSync(resourceSrcPath)) {
-        let relativeResourcePath = path.relative(
-          projectFilesDestPath,
-          resourceSrcPath
+  let appName = "ReactTestApp"; //Default value if manifest or 'name' field doesn't exist
+
+  if (manifestFilePath) {
+    const content = fs.readFileSync(manifestFilePath);
+    let resourcesPaths = {};
+    try {
+      const json = JSON.parse(content.toString());
+      resourcesPaths = json.resources;
+      appName = json.name || appName;
+    } catch (e) {
+      console.warn(`Couldn't parse \'app.json\': \n${e.message}`);
+    }
+    resourcesPaths =
+      (resourcesPaths && resourcesPaths.windows) || resourcesPaths;
+    if (Array.isArray(resourcesPaths)) {
+      for (const resource of resourcesPaths) {
+        const resourceSrcPath = path.relative(
+          path.dirname(manifestFilePath),
+          resource
         );
-        if (fs.statSync(resourceSrcPath).isDirectory()) {
-          relativeResourcePath = relativeResourcePath.concat("\\**\\*");
-          bundleDirContent = bundleDirContent.concat(
-            relativeResourcePath + ";"
+        if (fs.existsSync(resourceSrcPath)) {
+          let relativeResourcePath = path.relative(
+            projectFilesDestPath,
+            resourceSrcPath
           );
+          if (fs.statSync(resourceSrcPath).isDirectory()) {
+            relativeResourcePath = relativeResourcePath.concat("\\**\\*");
+            bundleDirContent = bundleDirContent.concat(
+              relativeResourcePath + ";"
+            );
+          } else {
+            bundleFileContent = bundleFileContent.concat(
+              relativeResourcePath + ";"
+            );
```

Comment:
Nit: `concat()` allows specifying more strings, so we should probably take advantage of that:
```suggestion
            bundleDirContent = bundleDirContent.concat(
              relativeResourcePath, "\\**\\*;"
            );
          } else {
            bundleFileContent = bundleFileContent.concat(
              relativeResourcePath, ";"
            );
```